test/Controllers/SearchTest.php

```diff
@@ -38,7 +36,7 @@ public function setUp() {
      * GonebusyLib\Controllers\SearchController::searchQuery()
      */
     public function testSearchQuery() {
-        $query = "testing@gonebusy.com"; // XXX Is this working?
```

Comment:
The only thing to notice though, is that this one comment I'm removing may indicate a bug in the search. I was guessing searching for that email address would return the Sandbox user but it doesn't. In fact I don't think search is returning users in general. May want to check that (: